src/Aspire.Dashboard/Utils/FilterHelpers.cs

```diff
@@ -0,0 +1,91 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Dashboard.Components.Layout;
+using Aspire.Dashboard.Components.Pages;
+using Aspire.Dashboard.Model;
+using Aspire.Dashboard.Model.Otlp;
+using Aspire.Dashboard.Resources;
+using Microsoft.Extensions.Localization;
+using Microsoft.FluentUI.AspNetCore.Components.Icons.Regular;
```

Comment:
nit: This is how we reference icons everywhere else.
```suggestion
using Icons = Microsoft.FluentUI.AspNetCore.Components.Icons;
```